src/components/Logo/AssetLogo.tsx

```diff
@@ -0,0 +1,60 @@
+import { SupportedChainId } from 'constants/chains'
+import useTokenLogoSource from 'hooks/useAssetLogoSource'
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import { MissingImageLogo } from './BaseLogo'
+
+const LogoImage = styled.img<{ size: string }>`
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  background: radial-gradient(white 60%, #ffffff00 calc(70% + 1px));
+  border-radius: 50%;
+  -mox-box-shadow: 0 0 1px white;
```

Comment:
Did you intentionally change the box-shadow to white? It used to be black.

Reply:
I did change on purpose. A lot of the new logos are darker, and thought the contrast was helpful
<img width="410" alt="image" src="https://user-images.githubusercontent.com/39385577/202531052-771cfdfc-ee7c-44f4-bb9f-95e6a4907713.png">
versus the new:
<img width="525" alt="image" src="https://user-images.githubusercontent.com/39385577/202531129-4e4938fa-70a3-45d1-8252-f911b2b3083a.png">
IMO it's not really noticable on non-dark images but adds good contrast to the dark ones

I felt comfortable removing native vs non-native shadow because the difference wasn't really noticeable before:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/39385577/202531479-7d2bc8b2-e006-4a95-ac38-5dc51d73af39.png">
(this is current version on app.uniswap.org, w/ eth having a white shadow and others not)

I'll have Cao skim through this convo and see if he has any issues w/ the design choices I made